frontend/src/views/RegisterPage.vue

```diff
@@ -140,13 +158,13 @@ export default {
             password: this.password,
             first_name: this.firstName,
             last_name: this.lastName,
-            is_teacher: 'false',
+            is_teacher: this.isTeacher ? 'true' : 'false',
             password_repeat: this.passwordConfirmation,
           })
               .then(() => {
                 this.errMessage = '';
                 this.termErrMessage = '';
-                this.$router.push(Feed);
+                this.$router.push({name: 'Feed'});
```

Comment:
Redirecting to feed without login is pointless. We could add automatic login upon registering to make this work, otherwise might as well redirect to the login page.

Reply:
Doesn't registering automatically login?  Because it wouldn't actually direct to Feed if youre not logged in (look at the router function in router.js)

Reply:
Yes, this was just changed in the backend to match this.